assets/component-list-menu.css

```diff
@@ -7,7 +7,6 @@
   min-width: 100%;
   width: 20rem;
   border: 1px solid rgba(var(--color-foreground), 0.2);
-  background-color: rgb(var(--color-background));
 }
 
```

Comment:
Not related to this PR but noticed this bug: https://screenshot.click/01-11-8oc29-vdpds.mp4

We are supposed to close one dropdown when another one opens, but if you start tabbing through it you are able to open the 2nd one without closing the first one

Reply:
Created an issue here: https://github.com/Shopify/dawn/issues/1435